packages/eddsa-poseidon/tests/index.test.ts

```diff
@@ -96,7 +96,7 @@ describe("EdDSAPoseidon", () => {
     it("Should sign a message (hexadecimal)", async () => {
         const message = "0x12"
 
-        const signature = signMessage(privateKey, message)
+        const signature = signMessage(privateKey, BigInt(message))
```

Comment:
This is a stringified bigint, so should be accepted as a BigNumber without explicit conversion.

I also suggest adding another test case with a decimal string like "123" which should also work.  Anything parsable by BigInt(s) should work.